src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php

```diff
@@ -641,6 +642,10 @@ protected function followRedirects($response)
      */
     protected function createTestResponse($response)
     {
-        return TestResponse::fromBaseResponse($response);
+        return tap(TestResponse::fromBaseResponse($response), function ($response) {
+            $response->withExceptions($this->app->bound(LoggedExceptionCollection::class)
+                            ? $this->app->make(LoggedExceptionCollection::class)
+                            : new LoggedExceptionCollection);
```

Comment:
It took me a sec to digest this. Try:

```suggestion
            $response->withExceptions(
                $this->app->bound(LoggedExceptionCollection::class)
                    ? $this->app->make(LoggedExceptionCollection::class)
                    : new LoggedExceptionCollection
            );
```